docker-compose.yml

```diff
@@ -20,8 +24,34 @@ services:
     networks:
       - backend
 
+  mysql:
+    image: mysql
+    command: --default-authentication-plugin=mysql_native_password
+    environment:
+      - MYSQL_ROOT_PASSWORD=natlas-dev-password-do-not-use
+      - MYSQL_DATABASE=natlas
+    networks:
+      - backend
+
+  zipkin-server:
+    image: openzipkin/zipkin-slim
+    ports:
+      - 9411:9411
+    networks:
+      - backend
+
+  opencensus-agent:
+    image: omnition/opencensus-agent
+    command: --config=/conf/opencensus-dev.yml
+    volumes:
+      - type: bind
+        source: ./.opencensus-dev.yml
+        target: /conf/opencensus-dev.yml
```

Comment:
We should mount this read-only since the agent never needs to modify it. Why do we need the explicit format? Does this not work:
```
volumes:
- ./.opencensus-dev.yml:/conf/opencensus-dev.yml:ro
```